### PR TITLE
Add a preview toot to the top of the display settings page.

### PR DESCRIPTION
**Pull request by xurble**
It also makes a few changes to the preview toot to accommodate them.

The (optional, default false) parse markdown flag stops the preview toot having weird highlighted regions when used as a placeholder, but allows links to look correct in settings.

### CODE REVIEW

File: IceCubesApp/Resources/Localization/ja.lproj/Localizable.strings [hunk 1]
```diff
@@ -66,6 +66,7 @@
 "settings.app.source" = "ソース (GitHub link)";
 "settings.app.support" = "アプリを支援する";
 "settings.app.about" = "アプリについて";
+"settings.display.example-toot" = "Example Toot";
```
Comment (nekolaboratory) [inline IceCubesApp/Resources/Localization/ja.lproj/Localizable.strings:69]: The best Japanese translation for this is:
"settings.display.example-toot" = "トゥートの例文";

File: IceCubesApp/Resources/Localization/nl.lproj/Localizable.strings [hunk 2]
```diff
@@ -66,6 +66,7 @@
 "settings.app.source" = "Broncode (GitHub-link)";
 "settings.app.support" = "De app steunen";
 "settings.app.about" = "Info";
+"settings.display.example-toot" = "Example Toot";
```
Comment (icod) [inline IceCubesApp/Resources/Localization/nl.lproj/Localizable.strings:69]:
```suggestion
"settings.display.example-toot" = "Postvoorbeeld";
```

**Comment by EvilOne** [comment]
Nice! I was going to work on this next—a few comments about the UI.

**Comment by EvilOne** [comment]
CleanShot 2023-02-08 at 05 40 34@2x

**Comment by EvilOne** [comment]
CleanShot 2023-02-08 at 05 42 36@2x

**Comment by EvilOne** [comment]
CleanShot 2023-02-08 at 05 44 21@2x

The preview should stay in place. This will allow smaller phones to make changes further down the screen and still see what those changes would do.

**Comment by EvilOne** [comment]
CleanShot 2023-02-08 at 05 47 08@2x

Highly recommend showing the share icon here to get rid of the "whitespace" and make it look more uniform.

**Comment by EvilOne** [comment]
Again, I like what I see. I'm sorry if my suggestions don't come off as suggestions. I do mean them as such.

**Comment by EvilOne** [comment]
How about a more personalised post example? I don't know what to put, but I like what Ivory did.

image

**Comment by Dimillian** [comment]
Thanks for the review @EvilOne I agree with all of those. @xurble I'm fixing some issue in order to merge it for next release, because this is a very good start! We can iterate on this.

**Comment by xurble** [comment]
I agree too, I've just been at work  all day 😬

**Comment by Dimillian** [comment]
No problem, I've just changed the wording and disabled the interactions which was necessary. More polish can come later :)

**Pull request merged**